src/modules/esl-utils/misc/object.ts

```diff
@@ -40,6 +43,16 @@ export function copyDefinedKeys<T>(obj?: T): Partial<T> {
   return result;
 }
 
+/** Makes a copy of obj with properties satisfying the predicate */
+export function omit(obj: any, predicate: (key: string, value?: any) => boolean): Record<string, any> {
```

Comment:
There is also no need to define value: any as optional because it will be available anyway